src/components/SignupsSplit.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from './ButtonLink';
+import ButtonGroup from './ButtonGroup';
+import { Link } from 'react-router-dom';
+import { useTreatments, SplitContext } from '@splitsoftware/splitio-react';
+
+const SignupsSplit = (props) => {
+  const treatments = useTreatments(['signups']);
+  const treatment = treatments['singups'];
+  const { isReady } = React.useContext(SplitContext);
+
+  const btns = (
+    <div {...props}>
+      <ButtonGroup>
+        <ButtonLink
+          data-tessen="stitchedPathLinkClick"
+          role="button"
+          to="https://newrelic.com/signup"
+          variant="primary"
+        >
+          Get an account
+        </ButtonLink>
```

Comment:
Only a suggestion but seeing as we are duplicating a couple buttons here, we could break those out into their own variables and just include them in each group. So we'd have:

```js
const accountBtn = (
       <ButtonLink
          data-tessen="stitchedPathLinkClick"
          role="button"
          to="https://newrelic.com/signup"
          variant="primary"
        >
          Get an account
        </ButtonLink>
)
```

and then include it using `{accountBtn}` in the `btn` group

Reply:
Thanks! I'll create it 🙌🏻